dom/attr/attr.js

```diff
@@ -344,6 +344,14 @@ var formElements = {"INPUT": true, "TEXTAREA": true, "SELECT": true},
 				},
 				set: function(values){
 					values = values || [];
+
+          // capture current selectedOptions
```

Comment:
Looks like a tabs v spaces (fight!) issue here. Might want to install an editorconfig plugin in your editor.